javaslang/generator/Generator.scala

```diff
@@ -370,6 +431,20 @@ def generateMainClasses(): Unit = {
               return $CharSeqType.of(sequence);
           }
 
+          ${genTraversableAliases(ArrayType, "array")}
+
+          ${genTraversableAliases(VectorType, "vector")}
+
+          ${genTraversableAliases(ListType, "list")}
+
+          ${genTraversableAliases(StreamType, "stream")}
+
+          ${genTraversableAliases(QueueType, "queue")}
+
+          ${genTraversableAliases(LinkedHashSetType, "linkedHashSet")}
+
+          ${genTraversableAliases(HashSetType, "hashSet")}
```

Comment:
@danieldietrich what aliases we need for access for `LinkedHashSet`, `HashSet`, `LinkedHashMap`, `HashMap`?
1. `linkedHashSet`, `hashSet`, `linkedHashMap`, `hashMap`
2. `linkedSet`, `set`, `linkedMap`, `map`